dashboard/src/pages/[workspaceSlug]/[appSlug]/logs/index.tsx

```diff
@@ -25,6 +25,7 @@ export default function LogsPage() {
     AvailableLogsServices.ALL,
   );
 
+  // create a client that sends http requests to Hausra but websocket requests to Bragi
```

Comment:
```suggestion
  // create a client that sends http requests to Hasura but websocket requests to Bragi
```

There are some comments in the function itself e.g. https://github.com/nhost/nhost/blob/eb3ba21afca6690ea173eba54d550f59d6a00e0c/dashboard/src/hooks/useRemoteApplicationGQLClientWithSubscriptions.tsx#L8-L13

though this appear on hover in an IDE

Reply:
this is more like an inline comment on a variable, so it's fine IMO